web/src/components/planner/PlannerFactory.vue

```diff
@@ -166,10 +174,10 @@
   import { Factory } from '@/interfaces/planner/FactoryInterface'
   import { DataInterface } from '@/interfaces/DataInterface'
   import { getPartDisplayName } from '@/utils/helpers'
-  import { formatNumber } from '@/utils/numberFormatter'
```

Comment:
If this strips out this import, it will break the page, as the `formatNumber` function requires it. This needs to stay.

Reply:
Alright, doing the fix quickly during work wasn't the best idea :P I'll have a look after/before work tomorrow

Reply:
@jeroenpol To assist future collaboration, PRs will now actually build the project's components so these issues are more apparent at dev time.